cli/src/pcluster/config/cluster_config.py

```diff
@@ -2401,6 +2418,22 @@ def _register_validators(self):
                         instance_type=compute_resource.instance_type,
                         instance_type_data=instance_types_data[compute_resource.instance_type],
                     )
+                # The validation below has to be in cluster config class instead of queue class
```

Comment:
Can line 2382 and these two validators to be added under CommonSchedulerClusterConfig? The reason is that from the schema, it seems that it also support for scheduler plugin schema

Reply:
CapacityReservationValidator and CapacityReservationResourceGroupValidator.
I think the field `capacity_reservation_target` is both for slurm and scheduler plugin in the schema. 
Why are the validators added for only class SlurmClusterConfig instaed of its parent class?
`class SlurmClusterConfig(CommonSchedulerClusterConfig)`

Reply:
`capacity_reservation_target` is only supported for Slurm in the schema